spec/lib/tasks_helpers/metrics_spec.rb

```diff
@@ -0,0 +1,102 @@
+describe 'Metrics' do
+  let(:header) do
+    %w[
+      tenant
+      portfolio_count
+      product_count
+      portfolio_share_count
+    ].collect(&:titleize)
+  end
+
+  let(:tenant) do
+    FactoryBot.create(
+      :tenant,
+      :name            => 'tenant_portfolio_shared_context',
+      :external_tenant => 'external_tenant_portfolio_shared'
+    )
```

Comment:
I think we can just do `create(:tenant, :name => 'tenant_portfolio_shared_context', :external_tenant => 'external_tenant_portfolio_shared')` here.